marketpulse/main/urls.py

```diff
@@ -3,5 +3,6 @@
 
 
 urlpatterns = patterns(
-    url(r'^$', '{0}.views.main'.format(settings.PROJECT_NAME), name='main'),
+    '{0}.main.views'.format(settings.PROJECT_MODULE),
```

Comment:
Personally I prefer the full path of the view to be part of the url, in case there are for example generic views. Moreover, I find it more readable but since this is just a trivial change it's totally up to you whether you change this or not.